src/main/java/mate/academy/spring/service/UserServiceImpl.java

```diff
@@ -22,4 +22,9 @@ public void add(User user) {
     public List<User> listUsers() {
         return userDao.getAll();
     }
+
+    @Override
+    public User get(Long id) {
+        return userDao.get(id).orElse(new User());
```

Comment:
returning a user and not throwing exception is not a good practice. if you are looking for some info and not getting it you need to know that something went wrong. here you won't know if something went wrong